src/_scss/pages/homepageUpdate/homepageUpdate.scss

```diff
@@ -6,8 +6,49 @@
   .homepage-content {
     .homepage-getting-started {
       height: 100%;
-      background-color: lightgrey;
+      background-color: white;
+      padding-bottom: $global-padding * 30;
+      overflow: hidden;
+      .homepage-background-flair {
+        position: absolute;
+        top: 5rem;
+        bottom: -50rem;
+        width: 125vw;
+        margin-left: calc(50% - 50vw);
+
+        background-color: $color-gray-warm-light;
+        opacity: 0.5;
+
+        @include transform(skewY(5deg));
+
+        @include media($medium-screen) {
```

Comment:
All of the places where we have @include media($medium-screen) I think these should be @include media($medium-screen - 1).  At the breakpoints, mainly 992px the images and text overlap but looks good at < 992px and > 992px
<img width="1022" alt="Screen Shot 2022-04-18 at 1 17 48 PM" src="https://user-images.githubusercontent.com/121010/163863506-235e1ed6-c395-4642-ad74-12de3994a9dc.png">
<img width="1022" alt="Screen Shot 2022-04-18 at 1 17 27 PM" src="https://user-images.githubusercontent.com/121010/163863551-678faadf-c231-4d9b-aaf0-8b316d388ad7.png">

Reply:
working on a fix for this breakpoint issue!